scripts/populate_faqs.py

```
@@ -0,0 +1,136 @@
+
+import sys
+import asyncio
+from pathlib import Path
+
+# Agregar el directorio raíz al path
+project_root = Path(__file__).parent.parent
+sys.path.insert(0, str(project_root))
+
+from app.services.embedding_service import embedding_service
+from app.db.config.database import get_async_session
+from app.db.models import FAQEmbedding
```

Ruff (0.12.2)

Module level import not at top of file

(E402)